src/components/ConfirmationDialog.jsx

```diff
@@ -0,0 +1,54 @@
+import Button from '@mui/material/Button';
+import Dialog from '@mui/material/Dialog';
+import DialogActions from '@mui/material/DialogActions';
+import DialogContent from '@mui/material/DialogContent';
+import DialogContentText from '@mui/material/DialogContentText';
+import DialogTitle from '@mui/material/DialogTitle';
+import { useState } from 'react';
+
+export default function AlertDialog(props) {
```

Comment:
Przerob to na komponent funkcyjny, a nie kopiuj 1:1 z dokumentacji MUI :)

Reply:
tyczy sie to calego komponentu- po co nam funkcje jak show i hide dialog jak mozemy to ograc useState'em

Reply:
to dalej nie jest zrobione :)